defender/tests.py

```diff
@@ -501,7 +501,6 @@ def test_admin(self):
         from .admin import AccessAttemptAdmin
         AccessAttemptAdmin
 
-    @patch('defender.config.PROTECTED_LOGINS', (ADMIN_LOGIN_URL, ))
     def test_decorator_middleware(self):
```

Comment:
I'm not sure @shin- added it I think. I think it tests the other middleware still, but I could be wrong. 

@shin- what do you think?